### PR TITLE
Complex documentation

### PR DESCRIPTION
**Pull request by cglzaguilar**

### CODE REVIEW

**Comment by cglzaguilar** [comment]
Everybody, please comment if you have any concern. Otherwise I'll rebase and merge this later today.

File: source/dox/Internals/API.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+\defgroup VIAA_PI VI Assembly Application Programming Interface (API)
```
Comment (spathiwa) [inline source/dox/Internals/API.md]: This looks like a typo of VIA_API

**Comment by spathiwa** [comment]
It seems weird that every prim gets its own \defgroup.

File: source/dox/Internals/API.md (hunk 1 above)
Comment (cglzaguilar) [inline source/dox/Internals/API.md]: Fixed on latest file changes.

**Comment by cglzaguilar** [comment]
Let's consider this an example of how the API might look like. There are several questions as to how best efficiently produce the documents and which tools to use. I'm open to suggestions.

**Pull request merged**